packages/varlet-ui/src/form/example/index.vue

```diff
@@ -1,22 +1,6 @@
 <script setup>
-import VarForm from '..'
-import VarInput from '../../input'
-import VarSelect from '../../select'
-import VarOption from '../../option'
-import VarCheckboxGroup from '../../checkbox-group'
-import VarCheckbox from '../../checkbox'
-import VarRadioGroup from '../../radio-group'
-import VarRadio from '../../radio'
-import VarButton from '../../button'
-import VarSwitch from '../../switch'
-import VarSlider from '../../slider'
-import VarUploader from '../../uploader'
-import VarCounter from '../../counter'
-import VarRate from '../../rate'
-import VarSpace from '../../space'
-import VarCustomFormComponent from './CustomFormComponent'
```

Comment:
Can't delete here

Reply:
`VarCustomFormComponent` not used, 
![image](https://github.com/varletjs/varlet/assets/81673017/b348bc4d-e2c0-4883-870c-434484b844e5)
other loaded normally.
![image](https://github.com/varletjs/varlet/assets/81673017/17a87ff1-1021-4d0b-ae21-30d317a0157d)